components/nugetAlias.txt

```diff
@@ -9,6 +9,7 @@ ASQFunctions: NServiceBus.AzureFunctions.StorageQueues
 ASP: NServiceBus.Persistence.AzureStorage
 ASTP: NServiceBus.Persistence.AzureTable
 Autofac: NServiceBus.Autofac
+BinaryDataBus: NServiceBus.DataBus.BinarySerializer
```

Comment:
Nitpick: I wonder if BinaryDataBusSerializer is a better name to not give the impression that this is some "binary storage option"

Reply:
Updated